server/sonar-server/src/main/java/org/sonar/server/project/ws/ProjectsWsSupport.java

```diff
@@ -50,4 +55,12 @@ OrganizationDto getOrganization(DbSession dbSession, String organizationKey) {
         dbClient.organizationDao().selectByKey(dbSession, organizationKey),
         "No organization for key '%s'", organizationKey);
   }
+
+  void checkCanUpdateProjectsVisibility(OrganizationDto organization, boolean newProjectsPrivate) {
+    try {
+      billingValidations.checkCanUpdateProjectVisibility(new BillingValidations.Organization(organization.getKey(), organization.getUuid()), newProjectsPrivate);
+    } catch (BillingValidations.BillingValidationsException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)